cmd/cmd.go

```diff
@@ -5,9 +5,11 @@ import (
 	"os"
 )
 
-// Bail handles exitable errors in commands.
+// BailOnError handles exitable errors in commands.
 // TODO: figure out what goes here.
-func Bail(err error) {
-	log.Println(err)
-	os.Exit(1)
+func BailOnError(err error) {
+	if err != nil {
+		log.Println(err)
```

Comment:
Since we are using the log pkg we can replace lines 12 and 13 with `log.Fatalln(err)`